guides/v2.1/javascript-dev-guide/widgets/widget_modal.md

```diff
@@ -181,7 +181,7 @@ The element that triggers the modal.
 
 ### `type` {#modal_type}
 
-The type of window: popup or slide.
+The type of window: popup.
```

Comment:
Hi @neeta-wagento 
Thanks for your contribution.
For the modal type is popup or slider, both options are available to make a modal.
so If we remove `slide` then user not seen the second `slide` option.